src/Aspire.Dashboard/README.md

```diff
@@ -0,0 +1,11 @@
+# .NET Aspire Dashboard
+
+## As a dotnet tool
+
+The dashboard can be packaged and executed as a `dotnet tool`.
```

Comment:
This reads more like documentation for ourselves, but I think this `README.md` will end up being used as the README for the package, so perhaps we need to review the content in that context?